Description: Remove zfssa from opts.py
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2016-01-02

Index: cinder/cinder/opts.py
===================================================================
--- cinder.orig/cinder/opts.py
+++ cinder/cinder/opts.py
@@ -158,10 +158,6 @@ from cinder.volume.drivers import vzstor
 from cinder.volume.drivers.windows import windows as \
     cinder_volume_drivers_windows_windows
 from cinder.volume.drivers import xio as cinder_volume_drivers_xio
-from cinder.volume.drivers.zfssa import zfssaiscsi as \
-    cinder_volume_drivers_zfssa_zfssaiscsi
-from cinder.volume.drivers.zfssa import zfssanfs as \
-    cinder_volume_drivers_zfssa_zfssanfs
 from cinder.volume import manager as cinder_volume_manager
 from cinder.wsgi import eventlet_server as cinder_wsgi_eventletserver
 from cinder.zonemanager.drivers.brocade import brcd_fabric_opts as \
@@ -267,7 +263,6 @@ def list_opts():
                 cinder_volume_drivers_xio.XIO_OPTS,
                 cinder_volume_drivers_ibm_storwize_svc_storwizesvcfc.
                 storwize_svc_fc_opts,
-                cinder_volume_drivers_zfssa_zfssaiscsi.ZFSSA_OPTS,
                 cinder_volume_driver.volume_opts,
                 cinder_volume_driver.iser_opts,
                 cinder_api_views_versions.versions_opts,
@@ -309,7 +304,6 @@ def list_opts():
                 cinder_volume_drivers_blockbridge.blockbridge_opts,
                 [cinder_scheduler_scheduleroptions.
                     scheduler_json_config_location_opt],
-                cinder_volume_drivers_zfssa_zfssanfs.ZFSSA_OPTS,
                 cinder_volume_drivers_disco_disco.disco_opts,
                 cinder_volume_drivers_hgst.hgst_opts,
                 cinder_message_api.messages_opts,
